datalake/backend/processing/check_metadata_content/task.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
```

Comment:
* I think it would be great to add at least some docstring documenting usage of this script

Reply:
The script is self-documenting:

```
$ datalake/backend/processing/check_stac_metadata/task.py
usage: task.py [-h] --metadata-url METADATA_URL
task.py: error: the following arguments are required: --metadata-url
```